src/components/pages/maintenance/jobDetails/jobDetails.js

```diff
@@ -51,7 +51,7 @@ export class JobDetails extends Component {
       .subscribe(
         jobStatus => this.setState({ jobStatus, jobStatusIsPending: false }),
         jobStatusError => this.setState({ jobStatusError, jobStatusIsPending: false }),
-      );
+    );
```

Comment:
nit: indentation got reset when it shouldn't have. It should line up with the function it belongs to.
```
  .subscribe(
    ..
  )
// other code
```